colour/models/rgb/ycbcr.py

```diff
@@ -0,0 +1,359 @@
+    'Rec.601' : [0.299, 0.114]
+    'Rec.709' : [0.2126, 0.0722]
+    'Rec.2020' : [0.2627, 0.0593]
+"""
+
+RANGE = (CaseInsensitiveMapping(
+        {'legal_12': [256./4095, 3760./4095],
+         'legal_12_YC': [256./4095, 3760./4095, 256./4095, 3840./4095],
+         'legal_12_int': [256, 3760],
+         'legal_12_YC_int': [256, 3760, 256, 3840],
+         'legal_10': [64./1023, 940./1023],
+         'legal_10_YC': [64./1023, 940./1023, 64./1023, 960./1023],
+         'legal_10_int': [64, 940],
+         'legal_10_YC_int': [64, 940, 64, 960],
+         'legal_8': [16./255, 235./255],
+         'legal_8_YC': [16./255, 235./255, 16./255, 240./255],
+         'legal_8_int': [16, 235],
+         'legal_8_YC_int': [16, 235, 16, 240],
+         'full': [0.0, 1.0],
+         'YPbPr': [0.0, 1.0, -0.5, 0.5]}))
+"""
+List of preset ranges:
+    'legal_12' : [256./4095, 3760./4095]
+    'legal_12_YC' : [256./4095, 3760./4095, 256./4095, 3840./4095]
+    'legal_12_int' : [256, 3760]
+    'legal_12_YC_int' : [256, 3760, 256, 3840]
+    'legal_10' : [64./1023, 940./1023]
+    'legal_10_YC' : [64./1023, 940./1023, 64./1023, 960./1023]
+    'legal_10_int' : [64, 940]
+    'legal_10_YC_int' : [64, 940, 64, 960]
+    'legal_8' : [16./255, 235./255]
+    'legal_8_YC' : [16./255, 235./255, 16./255, 240./255]
+    'legal_8_int' : [16, 235]
+    'legal_8_YC_int' : [16, 235, 16, 240]
+    'full' : [0.0, 1.0]
+    'YPbPr' : [0.0, 1.0, -0.5, 0.5]
+"""
+
+
+def rgb_to_YCbCr(rgb, K=WEIGHT['Rec.709'], inRange=RANGE['full'],
+                 outRange=RANGE['legal_10_YC']):
+    """
+    Converts an array of R'G'B' values to the corresponding Y'CbCr values.
+
+    Note: For ITU-R BT.2020 (Rec.2020) the rgb_to_YCbCr function is only
+    applicable tothe non-constant luminance implementation.
+    The rgb_to_YcCbcCrc function should be used for the constant luminance case
+    See https://www.itu.int/dms_pubrec/itu-r/rec/bt/\
+    R-REC-BT.2020-0-201208-S!!PDF-E.pdf
+
+    Parameters
+    ----------
+    rgb : array_like
+        Input R'G'B' values. These may be floats or integers.
+    K : array of two values
+        The luma weighting coefficients of red and blue.
+        Default: WEIGHT['Rec.709']
+    inRange : array of two values (optional)
+        The values of R', G' and B' corresponding to 0 IRE and 100 IRE.
+        These may be floats or integer code values.
+        Default: RANGE['full']
+    outRange : array of four values (optional)
+        The values of Y' corresponding to 0 IRE and 100 IRE
+        and the minimum and maximum values of Cb and Cr.
+        These may be floats or integer code values.
+        Default: RANGE['legal_10_YC']
+
+    Returns
+    -------
+    ndarray
+        Y'CbCr array.
+        These may be floats or integer code values, depending on the parameter
+        values given for outRange.
+
+    Examples
+    --------
+    >>> RGB = np.array([1.0, 1.0, 1.0])
+    >>> rgb_to_YCbCr(RGB)  # doctest: +ELLIPSIS
```

Comment:
`# doctest: +ELLIPSIS` is used to prevent doctests precision issues, we usually trim everything after the 7th decimal point, thus `array([ 0.91886608,  0.50048876,  0.50048876])` becomes `array([ 0.9188660...,  0.5004887...,  0.5004887...])`